### PR TITLE
Pickadiaper2

### PR DESCRIPTION
**Pull request by rmag42**

### CODE REVIEW

**Comment by exvertus** [comment]
Added https://trello.com/c/8nNOZtMx/17-make-pantheontsv-data-available-outside-the-repo-and-delete-it-from-github but this is fine/easiest for now

**Review by exvertus — APPROVED** [review]

File: app.js [hunk 1]
```diff
@@ -214,6 +223,46 @@ function right (msg) {
   imgPost(msg, youreRightPath);
 }
 
+// Asks you to pick a diaper
+async function pickADiaper (msg) {
+  const channelId = msg.channel.id;
+  await pickAPerson().then(async (line) => {
+    const person1Array = line.split('\t');
```
Comment (apkatsikas) [inline app.js:230]: could be nice to in-line top of the function define the various index positions, like `const countryIndex = 5;`

File: app.js [hunk 2]
```diff
@@ -214,6 +223,46 @@ function right (msg) {
...
+    }).catch(console.error);
+  }).catch(console.error);
+}
+
+function pickAPerson () {
+  let i = 0;
+  const randomInt = generateRandomInt(pickADiaperFileCount);
+  console.log('Random int is ' + randomInt);
+  return new Promise((resolve, reject) => {
+    lineReader.eachLine('pantheon.tsv', function (line) {
```
Comment (apkatsikas) [inline app.js:256]: maybe pantheon.tsv could be a const at the top of this file since it gets used twice

File: app.js [hunk 3]
```diff
@@ -229,12 +278,26 @@ function generateRandomInt (max) {
   );
 }
 
+function countDiaperFile () {
+  return new Promise((resolve, reject) => {
+    let i = 0;
+    lineReader.eachLine('pantheon.tsv', function (line) {
```
Comment (apkatsikas) [inline app.js:284]: pantheon const could be used here

**Review by apkatsikas — APPROVED** [review]
left some suggestions

**Comment by mat-lundin** [comment]
updated with Drew's suggestions

**Pull request merged**